pkg/crc/network/proxy.go

```diff
@@ -18,7 +18,7 @@ var (
 type ProxyConfig struct {
 	HTTPProxy   string
 	HTTPSProxy  string
-	noProxy     []string
+	NoProxy     []string
```

Comment:
Making this exportable means the consumer of `ProxyConfig` can make use of `NoProxy`? In the commit logs it says `user might want to verify` (which user in this context) ?

Reply:
This struct is exposed in the daemon API when asking for status or start. I thought it would be good to expose this also.

Reply:
@guillaumerose This info should be part of of logs to get the context :+1: